changelogs/unreleased/5036-relu-minor.md

```diff
@@ -0,0 +1,66 @@
+## HTTPProxy: implement HTTP query parameter matching
```

Comment:
This is a fantastic changelog, thank you!

Reply:
+1 from me